heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -113,7 +108,7 @@ TMaster::TMaster(const std::string& _zk_hostport, const std::string& _topology_n
   tmaster_location_->set_master_port(master_port_);
   tmaster_location_->set_stats_port(stats_port_);
   DCHECK(tmaster_location_->IsInitialized());
-  EstablishTMaster(EventLoop::TIMEOUT_EVENT);
+  EstablishPackingPlan(EventLoop::TIMEOUT_EVENT);
```

Comment:
If the intent is to not register the packing plan listener that handles updates (and physical plan deletion) until after we're fully up and running, that seems like a good call. We can differ the packing plan watch initialization until after we've become master, while still fetching the initial packing plan before registering location.